firmware/spade/src/rpi/main.c

```diff
@@ -213,6 +236,9 @@ int main() {
   jerry_init(JERRY_INIT_MEM_STATS);
   init(sprite_free_jerry_object); // TODO: document
 
+  // Initialize SD card
+  mount_sd_card();
```

Comment:
Something like this can almost surely fail?

Reply:
could you elaborate? all testing for failures is contained within the mount_sd_card function (which doesn't return anything), so I don't think anything should need to be checked here